src/vuex/modules/pushNotifications.js

```diff
@@ -66,25 +66,41 @@ const askPermissionAndRegister = async (activeNetworks, apollo) => {
     } catch (error) {
       console.log("Permission denied", error)
     }
+  } else {
+    // upsert
+    const token = localStorage.getItem("registration-push-notifications-token")
+    registerDevice(token, activeNetworks, apollo, true) // non-blocking
```

Comment:
So the logic is that every time a registered device logs in, the token does in fact change, or why do we need to update these infos?

Reply:
I've updated the flow a bit as this was not correct.

There are three options now:
1. device not registered === null
2. blocked push notifications === blocked
3. allowed push notifications === allowed

if null -> ask permission
if block -> skip flow
if allowed -> update device registration async when logging in (to detect new addresses so we can also create default topic subscriptions for those addresses/network).